src/mango/test/01-index-crud-test.py

```diff
@@ -122,8 +137,8 @@ def test_read_idx_doc(self):
             ddocid = idx["ddoc"]
             doc = self.db.open_doc(ddocid)
             self.assertEqual(doc["_id"], ddocid)
-            info = self.db.ddoc_info(ddocid)
-            self.assertEqual(info["name"], ddocid.split("_design/")[-1])
+            # info = self.db.ddoc_info(ddocid)
+            # self.assertEqual(info["name"], ddocid.split("_design/")[-1])
```

Comment:
Why are you disabling tests? Is this an incompatibility?

Reply:
we don't have the _info endpoint yet

Reply:
Why not add it? Seems relatively trivial compared to everything else?

Reply:
This has been merged to master so these tests can be fixed now.